app/models/programme.rb

```diff
@@ -45,6 +45,9 @@ def short_name
     raise NotImplementedError
   end
 
+  def certificate_name
+  end
```

Comment:
This ought to raise NotImplementedError, as otherwise we could end up with a slug being displayed as ""